network/framework/src/counters.rs

```diff
@@ -39,22 +41,22 @@ const PRE_DIAL_LABEL: &str = "pre_dial";
 pub const SERIALIZATION_LABEL: &str = "serialization";
 pub const DESERIALIZATION_LABEL: &str = "deserialization";
 
-pub static APTOS_CONNECTIONS: Lazy<IntGaugeVec> = Lazy::new(|| {
-    register_int_gauge_vec!(
-        "aptos_connections",
-        "Number of current connections and their direction",
-        &["role_type", "network_id", "peer_id", "direction"]
+// previous APTOS_CONNECTIONS counter replaced by PeersAndMetadataGauge which translates PeersAndMetadata global singleton directly to Prometheus Metrics data
```

Comment:
Does it make sense for us to create a new dashboard containing the metrics for the new code? It's hard to track via the diff on what's been removed/broken/changed.

Reply:
There's a new section in the `network` dashboard for net2 stuff. We can add misc new graphs there.